queenie:
  gripper_controller:
    type: "effort_controllers/JointGroupPositionController"
    joints:
    - neck
    - palm_riser
    - palm_left_finger
    - palm_right_finger
    # - left_finger_to_tip
    # - right_finger_to_tip
  # arm_controller:
  #   type: "effort_controllers/JointPositionController"
  #   joint: palm_riser
  #   pid: {p: 10, i: 0, d: 5}

  # palm_revolute_controller:
  #   type: effort_controllers/JointPositionController
  #   joint: palm_revolute
  #   pid: {p: 100, i: 1.0, d: 0.0}
    
/queenie/gripper_controller:
  neck:
    pid: {p: 100, i: 0, d: 5}
  palm_riser:
    pid: {p: 1000, i: 0, d: 5}
  palm_left_finger:
    pid: {p: 100, i: 0, d: 5}
  palm_right_finger:
    pid: {p: 100, i: 0, d: 5}
  # left_finger_to_tip:
  #   pid: {p: 1.0, i: 0, d: 0.1}
  # right_finger_to_tip:
  #   pid: {p: 1.0, i: 0, d: 0.1}
# /queenie/arm_controller:
#   palm_riser:
#   pid: {p: 10, i: 0, d: 5}